core/src/components/checkbox/checkbox.scss

```diff
@@ -158,6 +156,99 @@ input {
   opacity: 0;
 }
 
+
+// Justify Content
+// ---------------------------------------------
+
+:host(.checkbox-justify-space-between) .checkbox-wrapper {
+  justify-content: space-between;
+}
+
+:host(.checkbox-justify-space-between.checkbox-label-placement-start) .checkbox-icon,
+:host(.checkbox-justify-space-between.checkbox-label-placement-fixed) .checkbox-icon {
+  @include margin(0, 0, 0, auto);
+}
```

Comment:
I recommend wrapping `.checkbox-icon` in a `.native-wrapper` element like the other new form controls have. This way you can apply margins to the icon container and not worry about interfering with the SVG itself.

From there, you can get rid of these `.checkbox-icon` selectors and apply the following CSS:

```
.native-wrapper {
  display: flex;

  align-items: center;
}
```

You can look at the Toggle CSS code to see how it was done there too.